Userland/DevTools/HackStudio/main.cpp

```diff
@@ -185,6 +185,11 @@ void set_current_editor_wrapper(RefPtr<EditorWrapper> wrapper)
     s_hack_studio_widget->set_current_editor_wrapper(wrapper);
 }
 
+void show_file_title_modified()
+{
+    s_hack_studio_widget->update_file_title_after_modified();
```

Comment:
Let's just call `update_current_editor_title()` and `update_window_title()`, I don't see a need for a wrapper function